### ISSUE TITLE
Customzing Profit and Loss Statement Report

### ISSUE THREAD

**Issue by ShashaQin** [frappe/erpnext#3548]
Hi,
Now Profit and Loss and Balance Sheet work on Accumulated Amount. I added another check box filter 'accumulate value' on this page to provide option to user by modifying financial_statements.js file. And I also modified the python code to show partial amount of the selected period and add a 'totals' column at the end when user doesn't choose accumulate value.
Now I have to implement hiding this check box when I choose 'Yearly' option in the periodicity list box. It will be a nice feature to hide one control based on the option of other controls on the report page.

**Comment by umairsy** [comment]
Hello,

Profit and Loss report in the Financial Analytics doesn't show accumulated amount, but only for the specific month. Check:

Accounts >> Main Reports >> Financial Statement

Select Profit and Loss for a report in the first filter.

**Comment by ShashaQin** [comment]
Hi, umairsy, thank you for replying. But my point is not accumulated amount because I already provided a check box for user to choose accumulating or not. My point is how to hide this check box based on the option in periodicity list box in js file. I already posted this issue on the forum a few days ago, one engineer replied me this feature is not possible now and suggested me pulled a feature request in here.
Any reply will be appreciated.
